flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/join/JoinITCase.scala

```diff
@@ -94,28 +89,19 @@ class JoinITCase(expectedJoinType: JoinType) extends BatchTestBase {
       ))
   }
 
-  @Ignore // TODO support lazy from source
   @Test
   def testLongHashJoinGenerator(): Unit = {
     if (expectedJoinType == HashJoin) {
-      val sink = (new CollectRowTableSink).configure(Array("c"), Array(Types.STRING))
-      tEnv.registerTableSink("collect", sink)
-      tEnv.insertInto("collect", tEnv.sqlQuery("SELECT c FROM SmallTable3, Table5 WHERE b = e"))
-      var haveTwoOp = false
-      env.getStreamGraph.getAllOperatorFactory.foreach(o =>
```

Comment:
This test is just for this...
We can just invoke `TableEnvironment.explain` to check "LongHashJoin".

Reply:
This is a ITcase, so I modify and compare the final result.
what's meaning of `this...`? ,I'm a little confused.

Reply:
This test is for testing long hash join, just a it case can not verify it.
I am OK to move it to plan test if it is work.